products/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'categories'
+
+class Product(models.Model):
+    name        = models.CharField(max_length=45)
+    price       = models.IntegerField()
+    release_at  = models.DateField()
+    sales_rate  = models.IntegerField()
+    description = models.TextField()
+    category    = models.ForeignKey('Category', on_delete=models.CASCADE)
+    color       = models.ForeignKey('Color', on_delete=models.SET_NULL, null=True)
+    season      = models.ManyToManyField('Season', through='Product_Season')
+    size        = models.ManyToManyField('Size', through='Product_Detail')
+
+    class Meta:
+        db_table = 'products'
+
+class Size(models.Model):
+    size = models.IntegerField()
```

Comment:
class 명과 변수 명 중복은 피해주세요 
```suggestion
    name = models.IntegerField()
```

Reply:
> class 명과 변수 명 중복은 피해주세요

반영하였습니다.